src/UnitConverter/Unit/Pressure/Millibar.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Pressure;
+
+use UnitConverter\Measure;
+use UnitConverter\Unit\{ AbstractUnit, UnitInterface };
+
+/**
+ * Millibar unit data class.
+ *
+ * @version 1.0.0
+ * @since 1.0.0
+ * @author Teun Willems
+ */
+class Millibar extends PressureUnit
+{
+    protected function configure () : void
+    {
+        $this
+            ->setName("bar")
+
+            ->setSymbol("bar")
+
+            ->setUnits(100000000)
```

Comment:
I am pretty sure you had the correct amount of units the first time (`1 millibar === 100 pascal`) 😃 .

![image](https://user-images.githubusercontent.com/18744334/31469448-264d9134-aea7-11e7-81d4-e9d1839b8da6.png)